contracts/Front/Challenges.sol

```diff
@@ -233,6 +233,10 @@ contract Challenges is Base {
 
     function isOnCooldown(address resolverId, bytes32 challengeId) private view returns (bool) {
         uint lastResolvedSequenceNumber = atlasStakeStore.getLastChallengeResolvedSequenceNumber(resolverId);
-        return lastResolvedSequenceNumber != 0 && lastResolvedSequenceNumber.add(getCooldown()) > getChallengeSequenceNumber(challengeId);
+
+        // solium-disable-next-line operator-whitespace
+        return getChallengeCreationTime(challengeId).add(config.COOLDOWN_TIMEOUT()) > time.currentTimestamp() &&
+        lastResolvedSequenceNumber != 0 &&
+        lastResolvedSequenceNumber.add(getCooldown()) > getChallengeSequenceNumber(challengeId);
```

Comment:
Jeeeeez, why do we have to put all these in one line? 😝 
```
if(timedOut) {
  return something;
}
if(sequenceNumberStuff) {
  return something;
}
return something;
```

Reply:
I would normally agree with @Anze1m, but in this case I don't. The return value is of type bool. So in the example above `something` would be `true` for all the if's, and false on the last. 

Instead, I would argue there should be a second method `challengeCooldownExpired` and both of them should be referenced when checking if a Atlas can resolve.